StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Embedded/EmbeddedPaymentMethodsView.swift

```diff
@@ -114,6 +129,7 @@ class EmbeddedPaymentMethodsView: UIView {
             if initialSelection == selection {
                 applePayRowButton.isSelected = true
                 self.selection = initialSelection
+                self.selectedRowButton = savedPaymentMethodButton
```

Comment:
I'm having trouble following this logic.. here's how i'm reading it:
1.  If the initial selection is apple pay, then..
  i. applePayRowButton should be selected
  ii. set the self.selection = initialSelection
  iii. set the selectedRowButton back to the savedPaymentMethodButton (but wait, in "i", we've set apple pay row button to be selected?)

Reply:
Good point, the applePayRowButton.isSelected = true is a bit repetitive and reads weird. I moved this below as just one line for all.